src/App.tsx

```diff
@@ -74,11 +60,7 @@
                 )}
 
                 {author && loaded && !hasError && posts.length > 0 && (
-                  <PostsList
-                    posts={posts}
-                    selectedPostId={selectedPost?.id}
-                    onPostSelected={setSelectedPost}
-                  />
+                  <PostsList />
                 )}
```

Comment:
Ensure that the `PostsList` component correctly handles the `posts` prop. If `PostsList` expects props, make sure they are passed correctly here.